### PR TITLE
count HTTP requests from traversals

### PR DESCRIPTION
**Pull request by jsteemann**
Related enterprise PR: https://github.com/arangodb/enterprise/pull/236

### CODE REVIEW

File: arangod/Cluster/ClusterEdgeCursor.h [hunk 1]
```diff
@@ -52,13 +52,17 @@ class ClusterEdgeCursor : public graph::EdgeCursor {
...
+  size_t httpRequests() const override { return _httpRequests; }
+
  private:
   std::vector<arangodb::velocypack::Slice> _edgeList;
 
   size_t _position;
   CollectionNameResolver const* _resolver;
   arangodb::graph::BaseOptions* _opts;
   arangodb::graph::ClusterTraverserCache* _cache;
+  size_t _httpRequests;
```
Comment (goedderz) [inline arangod/Cluster/ClusterEdgeCursor.h]: @jsteemann Maybe I've just missed it, but I don't see where you're increasing `_httpRequests`. In the constructors `fetchEdgesFromEngines` from `Cluster/ClusterMethods.cpp` is called, which seems to make one request per engine.

Comment (jsteemann) [reply arangod/Cluster/ClusterEdgeCursor.h]: Good catch!
I think the instance variable in `ClusterEdgeCursor.h` is currently not increased, so I am counting only requests for vertices but not for edges.
Will add it!

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr-linux/5260/

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3880/

**Pull request merged**